code/_onclick/hud/other_mobs.dm

```diff
@@ -1,3 +1,15 @@
+/mob/living/simple_animal/create_mob_hud()
+	hud_used = new /datum/hud/simple(src)
+
+/datum/hud/simple/New(mob/user)
```

Comment:
You forgot to change this one

Reply:
Change how?

Reply:
Change it to `simple_animal`. A plain `simple` I see being easily confused for other systems.